pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -157,6 +161,7 @@ func (alloc *allocateAction) Execute(ssn *framework.Session) {
 					}
 
 					assigned = true
+					metrics.UpdateTaskScheduleDuration(metrics.Duration(taskScheduleStart))
```

Comment:
Is this the right way to measure `kube_batch_task_scheduling_latency_microseconds` ?

Reply:
no, it is not the right way to measure the latency, the start time should be measured  at the point when the task(pod) observed by kube-batch.

Reply:
and it is the same to way of measuring the latency of job

Reply:
> the start time should be measured at the point when the task(pod) observed by kube-batch

+1

Maybe we can add more info into `TaskInfo`.

Reply:
I think we can add `CreationTimestamp` in TaskInfo, when kube-batch create task, deep copy timestamp to task. This timestamp would be the startTime.

We probably want to add endTime inside 
1. job_info.go#UpdateTaskStatus. If status is Binding, we measure latency. Any other status indicate success of scheduling?  I am not sure what's Pipeline. Should we also consider it?

Reply:
> I think we can add `CreationTimestamp` in TaskInfo ...

If so, how to handle scheduler crash case? Maybe we can use Pod's `CreationTimestamp` directly; it only include the additional time from apiserver to scheduler which should be short.